src/scripts.js

```diff
@@ -2,16 +2,12 @@ import './css/base.scss';
 import './css/styles.scss';
 import './index.js';
 
-import userData from './data/users';
-import activityData from './data/activity';
-import sleepData from './data/sleep';
-import hydrationData from './data/hydration';
+
+import fetchData from //fetch.js file to be created later//
```

Comment:
Looks like some of the import files were taken out due to where we are using them! This should be fine after some work is put into the appropriate files